javascript/packages/orchestrator/src/keys.ts

```diff
@@ -85,6 +92,23 @@ export async function generateKeystoreFiles(
     rate: node.accounts.ed_account.publicKey, // Equilibrium rate module
   };
 
+  node.keystore_key_types?.forEach((key_type: string) => {
+    console.log(
+      "!!!! node.keystore_key_types",
+      key_type,
+      node.keystore_key_types,
+    );
+    console.log(
+      "!!!! node.keystore_key_types",
+      DEFAULT_KEYSTORE_KEY_TYPES.includes(key_type),
+    );
+    if (DEFAULT_KEYSTORE_KEY_TYPES.includes(key_type))
+      keysHash[key_type] = default_keystore_key_types[key_type];
+  });
+
+  if (!keysHash || Object.keys(keysHash).length === 0)
```

Comment:
```keysHashes``` can't be ```null```/```undefined``` because it's defined above and it's a empty object.

```suggestion
  if (Object.keys(keysHashes).length === 0)
```